components/windows-service/hooks/install

```diff
@@ -0,0 +1,7 @@
+. {{pkg.path}}\bin\habitat.ps1
+
+if((Get-Service Habitat -ErrorAction SilentlyContinue) -ne $null) {
+    UnInstall-HabService
+}
```

Comment:
If the service already exists, we uninstall the old and install the new, correct?

Reply:
that is right.